packages/hdwallet-ledger/src/mayachain.ts

```diff
@@ -0,0 +1,84 @@
+import type { AccountData, AminoSignResponse, OfflineAminoSigner, StdSignDoc, StdTx } from "@cosmjs/amino";
+import { Secp256k1Signature } from "@cosmjs/crypto";
+import type { SignerData } from "@cosmjs/stargate";
+import * as core from "@shapeshiftoss/hdwallet-core";
+import { fromByteArray } from "base64-js";
+import PLazy from "p-lazy";
+
+import { handleError, LedgerTransport, stringifyKeysInOrder } from "./index";
+
+const protoTxBuilder = PLazy.from(() => import("@shapeshiftoss/proto-tx-builder"));
+
+export const mayachainGetAddress = async (
+  transport: LedgerTransport,
+  msg: core.MayachainGetAddress
+): Promise<string> => {
+  const res = await transport.call("Thorchain", "getAddress", msg.addressNList, "maya");
```

Comment:
💀 does make sense since this is a fork, but confused me for a sec!

Reply:
ha yeah... thorchain app, maya addy